sktm/executable.py

```diff
@@ -38,6 +38,7 @@ def setup_parser():
     parser.add_argument("--jurl", help="Jenkins URL")
     parser.add_argument("--jlogin", help="Jenkins login")
     parser.add_argument("--jpass", help="Jenkins password")
+    parser.add_argument("--jretry", help="Counter to retry Jenkins")
```

Comment:
You can add `type=int` keyword argument here and then don't need to cast the option to `int` later. Granted, the value read from the config file won't be `int`, so you can combine this comment with the one below (about the default value assignment in `load_config` and do the retyping there.

Can you also mention the default value in the help string please? A simple `(defaults to X)` at the end is enough.

Reply:
The default value is added to the help string.